modules/order/src/AvailabilityOrderRefreshProcessor.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+use Drupal\commerce\AvailabilityManagerInterface;
+use Drupal\commerce_order\Entity\OrderInterface;
+
+/**
+ * Provides an order refresh processor for availability.
+ */
+class AvailabilityOrderRefreshProcessor implements OrderRefreshProcessorInterface {
```

Comment:
AvailabilityOrderProcessor, OrderProcessorInterface. The refresh part is redundant.